Sources/Commands/Workspace.swift

```diff
@@ -582,20 +582,30 @@ public class Workspace {
         try updateCheckouts(with: updateResults)
         // If we're repinning, update the pins store.
         if repin {
-            // Create a dictionary of result for fast lookup.
```

Comment:
I'm tempted to say update with autopin on should be same as repin, I think it is what people will want (by default).